packages/is-valid-prop/src/index.ts

```diff
@@ -541,18 +541,16 @@ const allCssProperties = [
 
 const properties = new Map(allCssProperties.map((prop) => [prop, true]))
 
-function memo<T>(fn: (value: string) => T): (value: string) => T {
+function memoize<T>(fn: (value: string) => T): (value: string) => T {
```

Comment:
I'd rather keep it named `memo` as I'm relying on this name
by removing this memo function declaration and importing the one from `styled-system/helpers.mjs` instead
https://github.com/chakra-ui/panda/pull/1115/files#diff-6f3b94515acc3625211140aac67d30d11205932fd8414b8ffb97b6a2a5c8c997R32